action.yml

```diff
@@ -71,10 +71,10 @@ inputs:
     description: 'A value of tmt.context variable https://tmt.readthedocs.io/en/latest/spec/context.html, variables separated by ;'
     required: false
     default: ''
-  artifacts_url:
-    description: 'Baseurl where results can be found'
+  tf_scope:
+    description: 'Defines the scope of Testing Farm. Possible options are public and private'
     required: false
-    default: 'http://artifacts.osci.redhat.com/testing-farm'
+    default: 'public'
 outputs:
```

Comment:
request_url output should IMO also be changed to ` "${{ steps.artifacts_url.outputs.url }}/${{ steps.sched_test.outputs.req_id }}"`

Reply:
Sorry, I don't follow. I will look at this PR once again. Maybe I did another typo.

Reply:
What `request_url` are you talking about?

Reply:
this one:
```
 82   request_url:
 83     description: 'An url of a scheduled testing farm request'
 84     value: ${{ steps.sched_test.outputs.results_url }}
 ```
 But as I look at it again, I think that the current approach is better, than the one I proposed.

Reply:
I got it. You are right, but request_id is part of outputs already.